### DISCUSSION TITLE
Self generating api docs are no longer generating properly from the Exposed Schemas setting

### DISCUSSION THREAD

**Discussion by PizzaConsole** [supabase/supabase#16949] [Contribute to Supabase]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Update `Exposed Schemas` setting in api settings to expose another schema or hide the graphql schema
2. Go to the generated API docs and it will still show graphql docs and it will NOT show the new schema

## Expected behavior

when you add or remove a schema it should update the generated api docs.

## System information

supabase studio in the cloud and the latest docker image

**Comment by saltcod** [comment]
Note: also causes an error if you access the <>API sidebar from a table in a custom schema
![CleanShot 2023-04-11 at 11 13 19@2x](https://user-images.githubusercontent.com/105593/231182411-99213d6d-1a37-40f2-a93e-4ea1ef6f406f.png)

**Comment by Odradek** [comment]
Bug still present, this is the browser's console log:
NS_ERROR_NOT_IMPLEMENTED: Component returned failure code: 0x80004001 (NS_ERROR_NOT_IMPLEMENTED) [nsIWindowsShellService.isCurrentAppPinnedToTaskbarAsync]
    task resource:///modules/BrowserGlue.sys.mjs:2432
    _scheduleStartupIdleTasks resource:///modules/BrowserGlue.sys.mjs:2904
BrowserGlue.sys.mjs:2443:21
Structured Ingestion ping failure with error: error 4 PingCentre.jsm:149:17
1685086370327	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/20aaf0b5-6697-45dc-a9df-9f1ed22bd7c7/main/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370327	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 20aaf0b5-6697-45dc-a9df-9f1ed22bd7c7: {"isTrusted":true} Log.sys.mjs:722
1685086370327	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/5577a1c8-a9d2-411b-96d3-6123342a755a/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370327	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 5577a1c8-a9d2-411b-96d3-6123342a755a: {"isTrusted":true} Log.sys.mjs:722
1685086370328	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/af0be8af-1c90-4771-a049-29207f03bb1e/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370328	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping af0be8af-1c90-4771-a049-29207f03bb1e: {"isTrusted":true} Log.sys.mjs:722
1685086370332	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/5949afee-6d44-4b0f-832e-fa618ba004c4/sync/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370332	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 5949afee-6d44-4b0f-832e-fa618ba004c4: {"isTrusted":true} Log.sys.mjs:722
1685086370338	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/48c2a7a0-3684-4438-ad0e-e131ddad37bc/event/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370338	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 48c2a7a0-3684-4438-ad0e-e131ddad37bc: {"isTrusted":true} Log.sys.mjs:722
1685086370345	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/96fe9dea-1905-493e-b5b2-c4f7e9fc8ca8/sync/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370345	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 96fe9dea-1905-493e-b5b2-c4f7e9fc8ca8: {"isTrusted":true} Log.sys.mjs:722
1685086370348	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/be474c62-e195-4736-aaf9-fcdb68246d86/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370348	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping be474c62-e195-4736-aaf9-fcdb68246d86: {"isTrusted":true} Log.sys.mjs:722
1685086370352	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/5ede9679-39d5-4ad2-9a68-250448a61109/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370352	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 5ede9679-39d5-4ad2-9a68-250448a61109: {"isTrusted":true} Log.sys.mjs:722
1685086370356	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/0f53e993-d367-4a4b-b91c-0bf4888963e5/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370356	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 0f53e993-d367-4a4b-b91c-0bf4888963e5: {"isTrusted":true} Log.sys.mjs:722
1685086370362	Toolkit.Telemetry	ERROR	TelemetrySend::_doPing - error making request to https://incoming.telemetry.mozilla.org/submit/telemetry/29236bb8-0dd1-41b1-831f-cd9a920b2a60/health/Firefox/113.0/release/20230504192738?v=4: eUnreachable Log.sys.mjs:722
1685086370362	Toolkit.Telemetry	ERROR	TelemetrySend::sendPersistedPings - failed to send ping 29236bb8-0dd1-41b1-831f-cd9a920b2a60: {"isTrusted":true} Log.sys.mjs:722
this.window.gBrowserInit is undefined ext-browser.js:1134
this.window.gBrowserInit is undefined ext-browser.js:1134

**Comment by PizzaConsole** [comment]
@saltcod Why did this get moved out of a bug?
